src/i18n/en-actions.ts

```diff
@@ -6,19 +6,27 @@ const actionsMessageDescriptors = {
   'action.title': `{actionType, select,
       ${ColonyActions.WrongColony} {Unknown Action}
       ${ColonyActions.Payment} {Pay {recipient} {amount} {tokenSymbol}}
+      ${ColonyMotions.PaymentMotion} {Pay {recipient} {amount} {tokenSymbol}}
       ${ColonyActions.MoveFunds} {Move {amount} {tokenSymbol} from {fromDomain} to {toDomain}}
+      ${ColonyMotions.MoveFundsMotion} {Move {amount} {tokenSymbol} from {fromDomain} to {toDomain}}
       ${ColonyActions.MintTokens} {Mint {amount} {tokenSymbol}}
       ${ColonyMotions.MintTokensMotion} {Mint {amount} {tokenSymbol}}
       ${ColonyActions.CreateDomain} {New team: {fromDomain}}
+      ${ColonyMotions.CreateDomainMotion} {New team: {fromDomain}}
       ${ColonyActions.VersionUpgrade} {Upgrade Colony to Version {newVersion}!}
       ${ColonyActions.ColonyEdit} {Colony details changed}
+      ${ColonyMotions.ColonyEditMotion} {Change colony details}
```

Comment:
It's a nitpick but why the copies of action and motion for edit colony different? Shouldn't they be the same?

Reply:
For edit domain and edit colony it doesn't sit well with me to use the original because they are in the past. 

You create an action and the change is instantly done, the details were _changed_ or the colony was _edited_. It's rightfully so in the past, the change/edit was already made the moment you created the action.

But with the motion versions, you create a motion to _edit_ domain or colony, you haven't edited the domain or colony until and _if_ the motion passes, so for me it doesn't make sense to use "changed" or "edited" like in the action version because the change hasn't been made.

This doesn't happen with the other action/motion titles because the others aren't in the past, only these 2.

Does that make sense to you? Or am I crazy? :D

Reply:
I didn't think about it initially, but it makes sense